src/renderer/components/deposit/add/SymDeposit.tsx

```diff
@@ -676,6 +678,25 @@ export const SymDeposit: React.FC<Props> = (props) => {
     )
   }, [asset, isApprovedERC20Token$, needApprovement, oPoolRouter, subscribeIsApprovedState])
 
+  const reloadFeesCheck = useCallback(() => {
+    if (!isEthChain(asset.chain)) {
+      reloadFeesHandler()
+    } else {
+      FP.pipe(
+        oPoolRouter,
+        O.map((router) => {
+          // ETH requires router to be set
+          if (router) {
+            prevAsset.current = O.some(asset)
+            reloadFeesHandler()
```

Comment:
You can call `reloadFeesHandler` which every change of selected asset. Check `LoadDepositFeesHandler`, it expects a O.Option<Address>` for `recipient`